### ISSUE TITLE
[Unit 4] easy check for each principle?

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-curricula#103]
> Homework Ideas…
> * Ask students to select one accessibility check for each of the principles they learned in previous topics and explain why the check falls under the principle.

Does Easy Checks include a check for each of the Principles?

_(I didn't have the bandwidth to check.)_

**Comment by daniel-montalvo** [comment]
Thanks for this!

I would say it includes checks for Perceivable, Operable and Understandable. Maybe the overall structure one can be also attached to the Robust principle but definitely it is not a clear association.
I will reword this as well.

**Comment by daniel-montalvo** [comment]
Closing this. Feele free to reopen if necessary. Work at 99502f153766bc3fdf1d6a1f25e52c05326b3f31
